### ISSUE TITLE
Publish New Version

### ISSUE THREAD

**Issue by foxxyz** [sandeepmistry/node-powermate#14]
Can you please publish a new version to NPM? Last release was 2 years ago, would be nice to get some of the updates since then (E.G. `disconnected` event on disconnect).

Thanks!

**Comment by brunoimbrizi** [comment]
I was just about to open an issue regarding the disconnect and I noticed it has already been addressed, but the fix hasn't been published to npm yet. Can you `npm publish` please?

**Comment by foxxyz** [comment]
In addition, looks like `node-hid` dependency breaks using Node v10 on OSX. 

```
 LIBTOOL-STATIC Release/hidapi.a
  CXX(target) Release/obj.target/HID/src/HID.o
../src/HID.cc:207:45: error: no matching member function for call to 'NewInstance'
    Local<Object> buf = nodeBufConstructor->NewInstance(1, nodeBufferArgs);
                        ~~~~~~~~~~~~~~~~~~~~^~~~~~~~~~~
```

Version in `package.json` is `0.5.x`, [current version is `0.7.2`](https://github.com/node-hid/node-hid/blob/master/package.json)

Perhaps this can be updated also.

**Comment by sandeepmistry** [comment]
@foxxyz @brunoimbrizi I've published v0.1.0 to npm now.

@foxxyz for the `node-hid` dependency bump, please try it out and then submit a pull request if everything looks ok.

**Comment by foxxyz** [comment]
@sandeepmistry Thank you! Pull request ready at #15
